content/styles/styleguide.css

```diff
@@ -22,6 +24,19 @@
   position: relative;
 }
 
+.st-colors,
+.st-buttons {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.st-full {
```

Comment:
why not just applying `width:100%` on `.st-main` and `.st-container`?

Reply:
Because I don’t want it. Some specific container need it but it regular time not